src/Builders/FilterBuilder.php

```diff
@@ -44,7 +44,7 @@ public function where($field, $value)
                 $this->wheres['must'][] = ['range' => [$field => ['gt' => $value]]];
                 break;
 
-            case '<';
+            case '<':
```

Comment:
Wow, what a terrible mistyping. I'm surprised that tests passed.

Reply:
Hey, no problems.. it happens ! :)